recipes/date/all/conanfile.py

```diff
@@ -1,29 +1,84 @@
 import os
-from conans import ConanFile, tools
+from conans import ConanFile, CMake, tools
 
 
 class DateConan(ConanFile):
     name = "date"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/HowardHinnant/date"
     description = "A date and time library based on the C++11/14/17 <chrono> header"
-    topics = ("date", "datetime", "timezone", "calendar", "time", "iana-database")
+    topics = ("date", "datetime", "timezone",
+              "calendar", "time", "iana-database")
     license = "MIT"
-    exports_sources = ["patches/*"]
-    no_copy_source = True
+    exports_sources = ["patches/*", "CMakeLists.txt"]
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package"
+    options = {"shared": [True, False],
```

Comment:
Could there still be a header-only version?

Reply:
Header only *will not* include tz library. 

i'll add that option as this was header only before.
option: header_only_no_tz           ( can strip the suffix _no_tz )
default: False  ( Maybe it hsould be default: True if we want to be backwards compatible ) .

Thoughts?

Reply:
Okay I am resolving this.

I think header only version of this library was basically a hack.
Original library' cmake does not provide a header only, yes it has some header only components but still a library in total is not header only.
Yes - it would be possible for someone to copy headers into their project and have a header only library - but I dont think thats an argument here.

Its still possible fairly easily actually i've looked at the code and it would work:
1) 
Copy all headers except tz.h ptz.h tz_private.h 

And you are done.

Now question is - what about options, should they be deleted? etc etc.